### PR TITLE
706 mexico izai

### LINKED ISSUE #706: Mexico IZAI
**Links to bulk download and/or API endpoints**
https://contratacionesabiertas.izai.org.mx/contratacionesabiertas/datosabiertos

**Link to CRM issue**
https://crm.open-contracting.org/issues/6062

**Priority**
e.g. anytime

### PR DESCRIPTION
**Pull request by cecicasco**
Closes #706

### CODE REVIEW

**Comment by yolile** [comment]
@cecicasco I think that this PR includes also the `739_mexico_quien_es_quien_records` branch changes, so can you start another one or remove those changes from this PR? thanks

**Comment by cecicasco** [comment]
> @cecicasco I think that this PR includes also the `739_mexico_quien_es_quien_records` branch changes, so can you start another one or remove those changes from this PR? thanks
 OK! I gonna check it.. thanks

File: kingfisher_scrapy/spiders/mexico_izai.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+      Instituto Zacatecano de Transparencia y Acceso a la Información (IZAI)
+    Spider arguments
+      from_date
+        Download only data from this year onward (YYYY format). Defaults to '2016'.
+      until_date
+        Download only data until this year (YYYY format). Defaults to '2021'.
+    API documentation
+      https://contratacionesabiertas.izai.org.mx/contratacionesabiertas/datosabiertos
+    """
+    name = 'mexico_izai'
```
Comment (yolile) [inline kingfisher_scrapy/spiders/mexico_izai.py:17]: let's name it as the other INAIs eg `mexico_zacatecas_izai`, also for the class name and file name, see `mexico_durango_idaip` for an example

**Review by yolile — CHANGES_REQUESTED** [review]

File: kingfisher_scrapy/spiders/mexico_zacatecas_izai.py [hunk 2]
```diff
@@ -2,7 +2,7 @@
 from kingfisher_scrapy.util import components
 
 
-class MexicoIZAI(PeriodicSpider):
+class MexicoZacatecasIZAI(PeriodicSpider):
```
Comment (yolile) [inline kingfisher_scrapy/spiders/mexico_zacatecas_izai.py:5]:
```suggestion
class MexicoZacatecas(PeriodicSpider):
```

File: docs/spiders.rst [hunk 3]
```diff
@@ -808,6 +801,13 @@ Mexico
 
    scrapy crawl mexico_yucatan_inaip
 
+.. autoclass:: kingfisher_scrapy.spiders.mexico_zacatecas_izai.MexicoZacatecasIZAI
```
Comment (yolile) [inline docs/spiders.rst:804]:
```suggestion
.. autoclass:: kingfisher_scrapy.spiders.mexico_zacatecas_izai.MexicoZacatecas
```

**Comment by yolile** [comment]
@cecicasco FYI the mexico changes are still in this branch and PR

**Pull request closed without merging**